client/src/styles/bootstrap-theme/wepickheroes.css

```diff
@@ -110,9 +110,9 @@ body {
   font-size: 1rem;
   font-weight: 400;
   line-height: 1.5;
-  color: #F4F3F3;
+  color: #062736;
   text-align: left;
-  background-color: #395155; }
+  background-color: #f4f3f3; }
```

Comment:
You don't actually want to change this css file. Instead change the `.scss` file. When you use `yarn start`, it will watch for changes to the `.scss` files and compile them to the `.css` files.